references/oidc-mock/apiproxy/policies/AM-SetLoginPage.xml

```diff
@@ -0,0 +1,161 @@
+        .login-submit {
+            border: 0px;
+            color: #fff;
+            text-shadow: 0 1px rgba(0,0,0,0.1);
+            background-color: #4d90fe;
+        }
+        .login-submit:hover {
+            border: 0px;
+            text-shadow: 0 1px rgba(0,0,0,0.3);
+            background-color: #357ae8;
+        }
+        .login-card a {
+            text-decoration: none;
+            color: #666;
+            font-weight: 400;
+            text-align: center;
+            display: inline-block;
+            opacity: 0.6;
+            transition: opacity ease 0.5s;
+        }
+        .login-card a:hover {
+            opacity: 1;
+        }
+        .login-help {
+            width: 100%;
+            text-align: center;
+            font-size: 12px;
+        }
+        .login-client-image img {
+            margin-bottom: 20px;
+        }
+        .login-card input[type=checkbox] {
+            margin-bottom: 10px;
+        }
+        .login-card label {
+            color: #999;
+            font-size: 16px;
+        }
+        .grant-debug {
+            font-family: Fixed, monospace;
+            width: 100%;
+            text-align: center;
+            font-size: 12px;
+            color: #999;
+            padding-top: 10px;
+        }
+        button {
+            cursor: pointer;
+        }
+    </style>
+</head>
+<body>
+<div class="login-card">
+    <h1>Login</h1>
+    <div class="login-client-image">
+       <img src="https://www.logo.wine/a/logo/Apigee/Apigee-Logo.wine.svg" width="50%">
+    </div>
+    <form autocomplete="off" action="/v1/openid-connect/consent" method="post">
+        <input required type="text" name="username" id="username" placeholder="Enter any login...">
+        <input required type="password" name="password" id="password" placeholder="...and password">
+        <input required type="hidden" name="state" value="{oidc.flow.authorize.state}">
+        <input required type="hidden" name="client_id" value="{oidc.flow.authorize.client_id}">
+        <input required type="hidden" name="redirect_uri" value="{oidc.flow.authorize.redirect_uri}">
+        <input required type="hidden" name="response_type" value="{oidc.flow.authorize.response_type}">
+        <input required type="hidden" name="scope" value="{oidc.flow.authorize.scope}">
+        <input required type="hidden" name="nonce" value="{oidc.flow.authorize.nonce}">
+        <input required type="hidden" name="prompt" value="{oidc.flow.authorize.prompt}">
+        <input required type="hidden" name="app_name" value="{oidc.flow.authorize.appName}">
+        <button type="submit" name="submit" id="submit" class="login login-submit">Sign-in</button>
+    </form>
+    <div class="login-help">
+        <a href="#">[ Cancel ]</a>
+        <a href="#">[ Terms of Service ]</a>
+        <a href="#">[ Privacy Policy ]</a>
+    </div>
+</div>
+</body>
+</html> ]]></Payload>
+    </Set>
+    <IgnoreUnresolvedVariables>true</IgnoreUnresolvedVariables>
```

Comment:
you have set this to true - what do you want to happen if any of the `oidc.flow.*` variables are missing for some reason?